lib/course.js

```diff
@@ -53,12 +53,18 @@ export async function getStudyGroup(groupSlug) {
   return { ...group }
 }
 
-export async function getCourse(course_id) {
+function normalizeSections(course, language) {
```

Comment:
This function should only get the course with all its metadata from the database (no language parameter). The language info should be decided only on the view component, not in the model.